README.md

```diff
@@ -22,7 +22,42 @@ This gem supports the following versions of Ruby and Rails:
 gem 'response_bank'
 ```
 
-2. use `#response_cache` method to any desired controller's action
+2. add an initializer file. We need to configure the `acquire_lock` method, set the cache store and the logger
+
+```ruby
+require 'response_bank'
+
+module ResponseBank
+  LOCK_TTL = 90
+
+  class << self
+    def acquire_lock(cache_key)
+      cache_store.write("#{cache_key}:lock", '1', unless_exist: true, expires_in: LOCK_TTL, raw: true)
+    end
+
+    def release_lock(cache_key)
+      cache_store.delete("#{cache_key}:lock")
+    end
+
+    def write_to_cache(cache_key)
```

Comment:
Only `acquire_lock` is expected to be overridden, so why are you also defining this method with a release lock? If we always need to match a acquire_lock with a release_lock we should always implement those methods in the gem, not asking all the apps to implement it.

Reply:
You are right, we only need `acquire_lock`, I updated the PR.